project.clj

```diff
@@ -121,6 +121,17 @@
              "-Djava.awt.headless=true"]                              ; prevent Java icon from randomly popping up in dock when running `lein ring server`
   :javac-options ["-target" "1.8", "-source" "1.8"]
   :uberjar-name "metabase.jar"
+  :uberjar-exclusions [
+                       #_#"\.clj(c)?$"
```

Comment:
so we can't actually remove all the Clojure source files because it looks like Clojure actually uses some of them at runtime